compile.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
 
 import subprocess
-subprocess.run(['pip-compile', 'requirements.in'])
-subprocess.run(['pip-compile', 'requirements-dev.in'])
+subprocess.run(['pip-compile', '-U', 'requirements.in'])
```

Comment:
I made this so that it actually updates the dependencies (if not pinned in `requirements.in`). I can't think of any case where I _just_ want to compile the deps without upgrading.